benchmark/opperf/utils/common_utils.py

```diff
@@ -108,12 +112,19 @@ def _prepare_op_benchmark_result(op, op_bench_result, profiler):
             inputs = value
         elif "avg_time" in key:
             avg_time = value
+        elif "p50_" in key:
```

Comment:
why not p50_time in key?

Reply:
coz it's enough to check if p50 exists in key
there won't be p50_a and p50_b

Reply:
Added it nonetheless